src/openforms/logging/logevent.py

```diff
@@ -201,7 +218,7 @@ def prefill_retrieve_success(submission: Submission, plugin, prefill_fields):
         "prefill_retrieve_success",
         extra_data={"prefill_fields": prefill_fields},
         plugin=plugin,
-        tags=[TimelineLogTags.AVG],
+        tags=[TimelineLogTags.submission_lifecycle],
```

Comment:
does the tag AVG need to be replaced or should be add the submission lifecycle tag to it? I'd expect the latter, because the fact that we retrieve personal details is an AVG event.